packages/components/src/components/dropdown-select/dropdown-select.tsx

```diff
@@ -208,6 +208,8 @@ export class DropdownSelect {
   @Prop() ariaLabelSelected?: string = 'selected';
   /** (optional) Text displayed in high contrast mode only to indicate disabled state */
   @Prop() hcmLabelDisabled?: string = 'this field is disabled';
+  /** (optional) is the element used in a form */
+  @Prop() formAssociated?: boolean = false;
```

Comment:
I wouldn't call this `formAssociated` because it's not exactly the same as [the standard static method](https://web.dev/articles/more-capable-form-controls).

I would actually not use any prop at all and just render the hidden input, always. The button has similar "hacks"… What do you think?

Reply:
makes sense, will do that instead